integration_test/lib/main.dart

```diff
@@ -433,25 +433,30 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future<void> testGetApptimizeTestInfo() async {
-    var result = await Apptimize.apptimizeTestInfo;
-
+    var result = await (Apptimize.apptimizeTestInfo
+        as FutureOr<Map<String, ApptimizeTestInfo?>?>);
+    assertNotNull(result);
+    if (result == null) return;
     assertFalse(result.isEmpty, "There were no result in apptimize test info");
 
     var firstEntry = result.entries.first;
-    assertEqual(firstEntry.key, firstEntry.value.testName,
+    assertEqual(firstEntry.key, firstEntry.value!.testName,
         "Key does not match test name");
     assertTrue(result.containsKey('FlutterFeatureFlagOn'),
         "Result expected to contain 'FlutterFeatureFlagOn'");
   }
 
   Future<void> testGetInstantUpdateAndWinnerInfo() async {
-    var result = await Apptimize.instantUpdateAndWinnerInfo;
+    var result = await (Apptimize.instantUpdateAndWinnerInfo
+        as FutureOr<Map<String, ApptimizeInstantUpdateOrWinnerInfo?>?>);
+    assertNotNull(result);
+    if (result == null) return;
```

Comment:
didn't the test stop at assert if it is null?

Reply:
It does stop; but yea, see my comment above.